woocommerce/woocommerce-android
# Issue/introduce app password tutorial UI

Summary
==========
Fix issue #11110 and #11000 by introducing the Application Password Tutorial view already using the Design reviews. 

Screenshots
==========
| Designs  | Actual |
| ------------- | ------------- |
| <img src="https://github.com/woocommerce/woocommerce-android/assets/5920403/61d2299e-d650-429b-ae63-ed186557ab40" width="270" height="570" /> | <img src="https://github.com/woocommerce/woocommerce-android/assets/5920403/c5dc48f2-4784-47f7-ad3c-2dce169d6b79" width="270" height="570" /> |

How to Test
==========
The UI is not connect to the app yet and can only be reached by using the `Run Preview` option from the Jetpack Compose toolset. Simply make sure the UI is aligned with the Figma designs.

Update release notes:
